src/themes/dark.scss

```diff
@@ -1,3 +1,9 @@
 .wxc-theme-dark {
   --wxc-meeting-background: linear-gradient(180deg, #1A1A1A 0%, #0F0F0F 100%);
+  // Icons
```

Comment:
thinking of a naming system, maybe:
--wxc-icon--audio-microphone--color
--wxc-icon--audio-microphone--active-color
--wxc-icon--camera--color
--wxc--icon-camera--active-color

active-color could also be named other-color or inner-color, maybe @lalli-flores can help us with the naming